app/src/main/java/com/framgia/forder/screen/searchproduct/ProductSearchResultFragment.java

```diff
@@ -30,9 +38,17 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
             @Nullable Bundle savedInstanceState) {
         List<Product> products = new ArrayList<>();
         ProductSearchResultAdapter adapter = new ProductSearchResultAdapter(getContext(), products);
-        mViewModel = new ProductSearchResultViewModel(adapter);
+        Navigator navigator = new Navigator(getParentFragment().getParentFragment());
+        mViewModel = new ProductSearchResultViewModel(adapter, navigator);
+
+        RealmApi realmApi = new RealmApi();
+        SharedPrefsApi prefsApi = new SharedPrefsImpl(getActivity());
+        ProductRepository productRepository = new ProductRepository(
+                new ProductRemoteDataSource(FOrderServiceClient.getInstance()),
```

Comment:
ERROR: [CORRECTNESS][NP_NULL_PARAM_DEREF_NONVIRTUAL] Non-virtual method call in com.framgia.forder.screen.searchproduct.ProductSearchResultFragment.onCreateView(LayoutInflater, ViewGroup, Bundle) passes null for non-null parameter of new com.framgia.forder.data.source.ProductRepository(ProductRemoteDataSource, ProductLocalDataSource, DomainRepository)
http://findbugs.sourceforge.net/bugDescriptions.html#NP_NULL_PARAM_DEREF_NONVIRTUAL